android-lib/src/androidTest/java/com/kinvey/androidTest/store/DataStoreTest.java

```diff
@@ -2567,6 +2568,53 @@ public void testSelfReferenceClassInClassWithList() throws InterruptedException
         assertNull(deleteCallback.getError());
     }
 
+// filed as MLIBZ-2647
+/*
```

Comment:
What is the reason for having these tests commented out?

Reply:
When I've added and run the test, the test was failed. After an investigation, I filed the bug `MLIBZ-2647`. But the test always is failed because of the bug `MLIBZ-2647`. So I commented it out.